README.md

```diff
@@ -71,6 +71,13 @@ Example usage:
 ./node_modules/.bin/ember-template-lint app/templates/application.hbs --json
 ```
 
+### ESLint
+
+If you're using [ember-inline-component](https://github.com/knownasilya/ember-inline-component)
```

Comment:
Lets make this a bit more generic here instead of focusing on ember-inline-component.  Perhaps:

> If you are using templates inlined into your JS files, you can leverage [eslint-plugin-hbs](https://github.com/psbanka/eslint-plugin-hbs) to integrate ember-template-lint into your normal eslint workflow.